development/dependency/python-probe-base/README.md

```diff
@@ -0,0 +1,24 @@
+# TMA-Monitor `Python` Client Library 
+
+Class files to use during the development of `TMA Monitor` probes developed in `Python`.
+
+
+## Prerequisites
+
+To use this library, you need to import in probe `Python` file the files presented in this directory and install `requests` module.
+
+There is also a `docker` image that contains all files and dependencies needed. In this case, `docker` is mandatory. 
+
+## Usage
+
+To use this library in the development of your probe, you just need to include the following lines in your probe main file, using the code below.
+
+```python
+from data import Data
+from message import Message
+from message import ComplexEncoder
+from observation import Observation
+from communication import Communication
+```
```

Comment:
Can't we add a module or something like that to add all of the classes at once?

Reply:
@joseadp That would be great. Can it be done?

Reply:
@nmsa , yes. Rui is working on it.

Reply:
@nmsa , Rui finished this library. Can you merge this PR, please?